### ISSUE TITLE
Backend: Subscribe and Unsubscribe a Community Page

### ISSUE THREAD

**Issue by OnurSefa** [bounswe/2021SpringGroup10#220]
A registered user should be able to subscribe and unsubscribe a community page. Lastly implemented endpoints are related to creation and updation of the community pages. But the subscription is not involved. Create related endpoints which take user id and community id to handle these operations

**Comment by OnurSefa** [comment]
I have tested subscribe functionality in several scenarios and the endpoint works just as we wanted. To achieve this, I have manipulated code in small amount of lines. The changes can be seen in the commit [OnurSefa/7543f24](https://github.com/bounswe/2021SpringGroup10/commit/7543f24913b776ccc4e76a0f2cb1c69ff2a6d058)

**Comment by OnurSefa** [comment]
I have tested unsubscribe operations manually and got successful results. So that, I decided to make pull request to enhance our develop branch with these new features. Also, implementation of these functionalities showed us that, an admin should be able to remove or accept subscriber requests in the private communities as mentioned in here #223 . And, if a user changes the privacy of the community from private to non private, all the subscription requesters should be automatically assigned as subscribers as mentioned in here #224. Because these mentioned functionalities are in different area than this issue, I created separate issues for them.
